test/dlc_tests/container_tests/bin/test_mx_dlc_telemetry_test.py

```diff
@@ -0,0 +1,81 @@
+import os
```

Comment:
nit: We generally use 4-space tab spacing for indentation in all other Python scripts. There is no rule that this indentation-scheme must be followed across all the Python scripts, but it would be good to have this be consistent in all newly added scripts in the future.

The python script file which was probably used as a template for this script will also have to be changed to follow this indentation scheme, which we will do at a later date.

Reply:
Addressed

Reply:
TF & PT indents fixed here: https://github.com/aws/deep-learning-containers/pull/305